package exception.com;

public class TestcustomException 
{
   static void validate(int age) throws TestException
   {
	   if(age<18)
	   {
		   throw new TestException("You are not eligible for vote");
	   }
	   else
	   {
		   System.out.println("you are not eligible for vote");
	   }
   }
	public static void main(String[] args)
	{
		
      try
      {
    	validate(22);
      }
        catch(TestException e)
      {
        	System.out.println(e);
      }
       finally
       {
    	   System.out.println("It always executed");
       }
        System.out.println("program will execute....");
	}

}
